packages/maskbook/src/plugins/Trader/apis/coinmarketcap/getPriceStat.ts

```diff
@@ -0,0 +1,127 @@
+import { CMC_LATEST_BASE_URL } from '../../constants/trending'
+import { EventEmitter } from 'events'
+
+const MAX_RETRY = 3
+let webSocket: WebSocket | null = null
+let retry = MAX_RETRY
+const Event = new EventEmitter()
```

Comment:
```suggestion
const events = new EventSource()
```

Reply:
No @septs EventSource is a completely different thing. https://developer.mozilla.org/en-US/docs/Web/API/EventSource

Do you mean `EventTarget`?

Reply:
ye